gufe/serialization/json.py

```diff
@@ -204,3 +209,147 @@ def serializer(self, obj: Any) -> str:
     def deserializer(self, string: str) -> Any:
         """Callable to loads JSON"""
         return self._deserializer(string)
+
+
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/gufe
+# custom_codecs.py: A place to keep various custom JSONCodec instances
+
+import datetime
```

Comment:
if we do keep this here, we probably shouldn't have imports half way through the file (and probably should change the header text above.

Reply:
Oops, yes of course. Fixing this now.